### PR TITLE
Use right repository in docker

### PR DESCRIPTION
**Pull request by tzimisce012**
Use 47deg user in docker

### CODE REVIEW

**Comment by JesusMtnez** [comment]
Should it be `higherkindness` user instead of `47deg`?

@juanpedromoreno

**Comment by juanpedromoreno** [comment]
@JesusMtnez As it's based on an environment variable, it's something we can tweak easily down the road.

File: build.sbt [hunk 1]
```diff
@@ -9,7 +9,7 @@ lazy val V = new {
   val betterMonadicFor: String = "0.3.0"
   val cats: String             = "1.6.0"
   val catsScalacheck: String   = "0.1.1"
-  val mouse: String            = "0.20"
+  val mouse: String            = "0.21"
```
Comment (juanpedromoreno) [inline build.sbt]: I think this might cause problems, I believe mouse 0.21 is for cats 2.0?

Comment (JesusMtnez) [reply build.sbt]: You are right. mouse should not be updated

**Pull request merged**